CHANGELOG.md

```diff
@@ -7,11 +7,34 @@ This projects adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 Up to version 8.0.0, the `major.minor` version numbers were based on the PHP version for which compatibility check support was added, with `patch` version numbers being specific to this library.
 From version 8.0.0 onwards, [Semantic Versioning](http://semver.org/) is used.
 
+**IMPORTANT**: The 8.0.0 release contains a **breaking change**. Please read the below information carefully before upgrading!
+
 
 ## [Unreleased]
 
 _Nothing yet._
 
+## [8.0.1] - 2017-08-07
+
+See all related issues and PRs in the [8.0.1 milestone].
+
+### Added
+- PHP 7.2 trailing comma in group use statement is now allowed
+- :fire: PHP 7.2 deprecated functions
```

Comment:
:star: `DeprecatedFunctions` sniff: recognize some more PHP 7.2 deprecated functions

(A few were already added in 7.1.4 + No guarantee as of yet that we're covering them all)